pkg/deviceplugin/api.go

```diff
@@ -30,6 +30,14 @@ type DeviceInfo struct {
 	topology *pluginapi.TopologyInfo
 }
 
+// UseDefaultAllocateMethodError allows the plugin to request running the default allocation
+// logic even while implementing the optional Allocator interface.
+type UseDefaultAllocateMethodError struct{}
```

Comment:
Let's rename it to `UseDefaultMethodError` to reuse it in future if we ever end up with other methods replacing the default ones.

Reply:
ok

Reply:
Done